main/options/options.cc

```diff
@@ -623,6 +624,14 @@ void readOptions(Options &opts,
             Exception::raise("simulated crash");
         }
 
+        if (raw.count("allowed-extension") > 0) {
+            auto exts = raw["allowed-extension"].as<vector<string>>();
```

Comment:
I would have expected `--allowed-extension` to have placed the given extension _on top of_ the `.rb` and `.rbi` extensions. I understand this is more flexible but is there any case where someone would not want to have `.rb` and `.rbi` extensions?

Reply:
Yes: me when testing and debugging :)

I sometimes even need to move / rename a RBI file to check things.

Reply:
This way felt more elegant than: `--no-default-extensions --allowed-extension .ru`.